pip_audit/_subprocess.py

```diff
@@ -52,9 +52,8 @@ def run(args: Sequence[str], *, log_stdout: bool = False, state: AuditState = Au
     # once `stdout` hits EOF, so we don't have to worry about that blocking.
     while not terminated:
         terminated = process.poll() is not None
-        # NOTE(ww): Buffer size chosen arbitrarily here and below.
-        stdout += process.stdout.read(4096)  # type: ignore
-        stderr += process.stderr.read(4096)  # type: ignore
+        stdout += process.stdout.read()  # type: ignore
+        stderr += process.stderr.read()  # type: ignore
```

Comment:
This is for a separate bug you're seeing, right? If so, I'd prefer to break this out into a separate issue and PR -- we've had trouble with this bit of code before, so I don't want to make a change here as part of a separate bugfix 🙂

Reply:
No problem, it's here now #756

Reply:
Thank you!